_pages/oidc/authorization.md

```diff
@@ -139,10 +143,36 @@ In an **unsuccessful authorization**, the URI will contain the parameters `error
         {% include accordion.html content=loa_values accordion_id="loa_accordion" title="Level of Assurance (LOA) Values (Deprecated)" id="loa_values" %}
       </div>
       <p id="fn:1">
-        1. Login.gov continues to work toward achieving certification of compliance with NIST’s IAL2 standard from a third-party assessment organization. 
+        1. Login.gov continues to work toward achieving certification of compliance with NIST’s IAL2 standard from a third-party assessment organization.
         <a href="#fnref:1">↩</a>1 <a href="#fnref:1:2">↩</a>2 <a href="#fnref:1:3">↩</a>3
       </p>
     </div>
+    <div class="grid-row dev-doc-row">
+      <div class="grid-col-5">
+        <h4 id="vtr" class="parameters">vtr</h4>
+      </div>
+      <div class="grid-col-7">
+          A JSON encoded array of Vectors of Trust used to specify the authentication and proofing features that are used to
+          authenticate and proof the user.  These and the <code class="language-plaintext highlighter-rouge">scope</code> determine which <a class="usa-link" href="{{ '/attributes/' | prepend: site.baseurl }}">user attributes</a> will be available in the <a class="usa-link" href="{{ '/oidc/user-info/#user-info-response' | prepend: site.baseurl }}">user info response</a>.
+        <p>
+          If the <code class="language-plaintext highlighter-rouge">vtr</code> parameter is passed then the
+          <code class="language-plaintext highlighter-rouge">vtm</code> parameter is required.
+        </p>
+      </div>
+    </div>
+    <div class="grid-row dev-doc-row">
+      <div class="grid-col-5">
+        <h4 id="vtr" class="parameters">vtm</h4>
+      </div>
+      <div class="grid-col-7">
+          A URL linking to the Trustmark for the <code class="language-plaintext highlighter-rouge">vtr</code> parameter.
+          This should be set to <a class="usa-link" href="{{ '/vot-trust-framework/' | prepend: site.baseurl }}">
+          https://developer.login.gov/vot-trust-framework</a>.
```

Comment:
This URL is intentionally not configured to use the baseurl here. It needs to match the expected VTM value that the IdP knows about: https://github.com/18F/identity-idp/blob/5837343cd6c1a9fe3f78d56d9931808e108e80a0/config/application.yml.default#L344

Reply:
Our canonical domain name for these docs is the plural one (developers), should we use that here (and in the VM claim?)

```suggestion
          https://developers.login.gov/vot-trust-framework</a>.
```